CMakeLists_files.cmake

```diff
@@ -25,6 +25,10 @@ list (APPEND PUBLIC_HEADER_FILES
         opm/output/vtk/writeVtkData.hpp
     )
 
+list (APPEND EXAMPLE_SOURCE_FILES
+		 applications/compareSummary.cpp
+)
+
```

Comment:
this is my fault. i forgot to mention that in order for an application to be marked for installation, it also needs to be added to the PROGRAM_SOURCE_FILES variable. that is, EXAMPLE_SOURCE_FILES is for being built, PROGRAM_SOURCE_FILES is for being installed.